ix-dev/community/bitcoind-knots/templates/docker-compose.yaml

```
@@ -58,7 +58,7 @@
   "-prune=%d" | format(values.bitcoin.prune_size * GB_TO_MIB),
   "-txindex=%d" | format(values.bitcoin.prune_size == 0 | int),
   "-datacarrier=%d" | format(values.bitcoin.op_return_txs_relay_enabled | int),
-  "-datacarriersize=%d" | format(values.bitcoin.max_op_return_size),
+  "-datacarriersize=%d" | format((values.bitcoin.max_op_return_size if values.bitcoin.max_op_return_size is defined else 0)),
```

```suggestion
  "-datacarriersize=%d" | format(values.bitcoin.max_op_return_size or 0),
```

I think this should work as well
in case of null, none, 0 or anything falsy it will be 0